dtslint/ts3.5/index.tsx

```diff
@@ -227,3 +227,58 @@ type RC = {
   b: ObservableQuery<number, void, number>;
 };
 type PC = ProductA<RC>; // $ExpectType Pick<{} & { a: string; b: number; }, "a" | "b">
+
+declare const q1: ObservableQuery<{ a: string }, string, string>;
+declare const q2: ObservableQuery<{ b: number }, string, string>;
+declare const q3: ObservableQuery<void, string, string>;
+
+<WithQueries
+  queries={{ q1, q2 }}
+  params={{ q: { a: 'ciao' }, q2: { b: 2 } }} // $ExpectError
+  render={q => {
+    return q.fold(
+      () => null,
+      () => null,
+      ({ q1, q2 }) => <span>{`${q1}: ${q2}`}</span>
+    );
+  }}
+/>;
+
+<WithQueries
+  queries={{ q1, q2 }}
+  params={{ q1: { a: 2 }, q2: { b: 2 } }} // $ExpectError
+  render={q => {
+    return q.fold(
+      () => null,
+      () => null,
+      ({ q1, q2 }) => <span>{`${q1}: ${q2}`}</span>
+    );
+  }}
+/>;
+
+<WithQueries
+  queries={{ q2 }}
+  params={{ q2: { b: 2 } }}
+  render={q => {
+    return q.fold(() => null, () => null, ({ q }) => <span>{q}</span>); // $ExpectError
+  }}
+/>;
+
+<WithQueries // $ExpectError
```

Comment:
what is the error here?

Reply:
because when you only define `queries` that have 0 arguments the `params` prop is set to `never`